CalibTracker/SiStripLorentzAngle/plugins/LorentzAngleRunInfo.cc

```diff
@@ -57,7 +57,7 @@ void LorentzAngleRunInfo::beginRun(const edm::Run & irun, const edm::EventSetup
    run = irun.run();
    edm::ESHandle<TrackerGeometry> theTrackerGeometry;         iSetup.get<TrackerDigiGeometryRecord>().get( theTrackerGeometry );  
    edm::ESHandle<MagneticField> magfield;                     iSetup.get<IdealMagneticFieldRecord>().get(magfield);                      
-   edm::ESHandle<SiStripLorentzAngle> SiStripLorentzAngle;    iSetup.get<SiStripLorentzAngleDepRcd>().get(SiStripLorentzAngle);      
+   edm::ESHandle<SiStripLorentzAngle> SiStripLorentzAngle;    iSetup.get<SiStripLorentzAngleRcd>().get("deconvolution",SiStripLorentzAngle);      
```

Comment:
Hi @robervalwalsh Not sure why we need to hard-code "deco" here.  Isn't it OK to consume the dependent record and let it deal with the latency switches?

Reply:
Hi @mmusich, I thought so, but the numbers coming out were different.

Reply:
Btw, the values from the dependent record were also different from the ones in the payload inspector.

Reply:
> I thought so, but the numbers coming out were different.

Because you were using some runs in peak?

Reply:
>  Btw, the values from the dependent record were also different from the ones in the payload inspector.

what run number did you check?

Reply:
I am pretty sure I was using a run in deco. I have to go through my notes to find it. May take a while